recipe/meta.yaml

```
@@ -15,7 +15,7 @@ package:
 
 source:
   url: https://github.com/h5py/h5py/archive/{{ version }}.tar.gz
-  sha256: 4937c1cbef049fd28973494def11084b70c1d218c6898a9d8cfb471c84010a99
+  sha256: d7d17c26da61b08489ce5c07cae0de1ed3fedfd2386a989e74652f111af6c171
```

```suggestion
  sha256: beb62edf226b8821d3e06f4d35f57055c9ac2aca11961574d505a8d6b7919192
```

The SHA of the tarbal from github can include the names of any branches that happen to be on the tag.

We should look at why we use GH instead of pypi, but this is the quick fix.